aten/src/ATen/native/native_functions.yaml

```diff
@@ -223,17 +223,22 @@
 - func: batch_norm(Tensor input, Tensor? weight, Tensor? bias, Tensor? running_mean, Tensor? running_var, bool training, double momentum, double eps, bool cudnn_enabled) -> Tensor
   variants: function
 
-- func: bernoulli(Tensor self, Tensor p, Generator* generator=nullptr) -> Tensor
```

Comment:
These two variants aren't documented or used, and they are more like tensor factory methods but not written according to the standard. So I deleted them.